build.sh

```diff
@@ -105,9 +105,17 @@ if (( ${NUMARGS} == 0 )) || hasArg libcudf; then
     make -j${PARALLEL_LEVEL} VERBOSE=${VERBOSE} ${INSTALL_TARGET}
 fi
 
-# Build and install the cudf Python package
+# Build and install the nvstrings and cudf Python packages
 if (( ${NUMARGS} == 0 )) || hasArg cudf; then
 
+    cd ${REPODIR}/python/nvstrings
+    if [[ ${INSTALL_TARGET} != "" ]]; then
+    python setup.py build_ext
+    python setup.py install --single-version-externally-managed --record=record.txt
+    else
+    python setup.py build_ext --library-dir=${LIBCUDF_BUILD_DIR}/nvstrings
```

Comment:
Why aren't you passing `--inplace` to the build_ext command here?

Reply:
`--inplace` is not an option passed to the Python build process for `custrings`. [Reference](https://github.com/rapidsai/custrings/blob/branch-0.10/build.sh#L112-L121).

Reply:
👍 still need to separate custrings into a separate build command instead of in the cudf / libcudf commands but understood.